usvg/src/convert/text/convert.rs

```diff
@@ -60,6 +60,7 @@ pub struct TextPath {
     pub start_offset: f64,
 
     pub path: tree::SharedPathData,
+    pub transform: Transform,
```

Comment:
I guess we can apply the transform immediately. No need to store the transform separately.

Reply:
Good point. Done!